skype/skype-ps/skype/Move-CsUser.md

```diff
@@ -14,7 +14,13 @@ ms.reviewer:
 
 ## SYNOPSIS
 
-Moves one or more user accounts enabled for Skype for Business Server to TeamsOnly (or the reverse). This cmdlet also can be used to move on-premises users from one pool to another.
+Moves one or more user accounts enabled for Skype for Business Server to TeamsOnly (or the reverse). This cmdlet also can be used to move on-premises users from one pool to another. 
+
+**PRE-REQUISITES**
+- Install or upgrade MicrosoftTeams powershell module with version >= 5.2.0
+- Run Set-TeamsEnvironmentConfig -TeamsEnvironmentName TeamsChina
```

Comment:
It is also better to provide link for set-teamsEnvironmentConfig docs
https://learn.microsoft.com/en-us/powershell/module/teams/set-teamsenvironmentconfig?view=teams-ps